specificatie/Waardelijsten/openapi.yaml

```diff
@@ -222,37 +223,71 @@ components:
       required: false
       schema:
         type: string
+        example: "AardZakelijkRecht"
     waardelijstidentificatie_path:
       name: waardelijstidentificatie
       in: path
       description: "De identificatie, behorende bij de waardelijst."
       required: true
       schema:
         type: string
+        example: "AardZakelijkRecht"
     code:
       name: code
       in: query
       description: "De code, behorende bij de waardelijstwaarde."
       required: false
       schema:
         type: string
+        example: "6"
     code_path:
       name: code
       in: path
       description: "De code, behorende bij de waardelijstwaarde."
       required: true
       schema:
         type: string
+        example: "6"
+    lijstomschrijving:
+      in: query
+      name: omschrijving
+      description: "De omschrijving (lange naam) van de waardelijst. <br> Bij het zoeken mag er gebruik worden gemaakt van <b>[wildcards](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/wildcard.feature)</b>.<br>Zoeken is <b>[case-Insensitive](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/case_insensitive.feature)</b></a>"
+      required: false
+      schema:
+        type: string
+        example: "aard zakelijk*"
     waarde:
       in: query
       name: waarde
       description: "De waarde uit de lijst. <br> Bij het zoeken mag er gebruik worden gemaakt van [wildcards](https://github.com/VNG-Realisatie/Haal-Centraaal-common/blob/master/features/wildcard.feature). <br> **Zoeken met tekstvelden** is <a href='https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/case_insensitive.feature'> <b>case-Insensitive </b></a>"
       required: false
       schema:
         type: string
+        example: "huurrecht*"
+    inclusiefbeeindigd:
+      in: query
+      name: inclusiefbeeindigd
```

Comment:
Ik neem aan dat we geen camelCase toepassen omdat het querystring parameter is

Reply:
Klopt.